tools/harness-simulation/posix/sniffer_sim/proto/sniffer.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package sniffer;
+
+// Sniffer simulation
+service Sniffer {
+    // Start the sniffer
+    rpc Start(StartRequest) returns (StatusReply) {}
+
+    // Let the sniffer sniff these nodes only
+    rpc FilterNodes(stream NodeID) returns (StatusReply) {}
```

Comment:
I don't see the advantage of using `stream` here. Why not define a request type containing `NodeID`s?

Reply:
Do you mean to use repeated fields instead? Something like the following codes?
```proto
message FilterNodesRequest {
    repeated int32 nodeids = 1;
}
```

Reply:
yes